### PR TITLE
Ensure compatibility with IDAES v2

### PR DESCRIPTION
**Pull request by lbianchi-lbl**
## Fixes/Addresses/Summary/Motivation:

- Check what works (or doesn't) when `idaes-pse` 2.x is used for installation
### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

**Review by ksbeattie — DISMISSED** [review]
LGTM

File: setup.py [hunk 1]
```diff
@@ -59,7 +59,7 @@
         "pandas==1.2.*",
         "openpyxl",
         # for the moment mainly for getting solvers with `idaes get-extensions`
-        "idaes-pse",
+        "idaes-pse==2.0.0rc0",
```
Comment (lbianchi-lbl) [inline setup.py:62]:
```suggestion
        # https://peps.python.org/pep-0440/#compatible-release
        "idaes-pse ~= 2.0",
```

**Comment by lbianchi-lbl** [comment]
We still need testing to ensure that Pyomo 6.5.0 does not negatively impact with the soon-to-be-added MINLP functionality needed for the March release.

**Comment by lbianchi-lbl** [comment]
I'm going ahead and merge this in since it won't affect whether Pyomo 6.5.0 or a different version is being installed.

**Pull request merged**